dappnode_package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "bitcoin.dnp.dappnode.eth",
-  "version": "0.1.6",
-  "upstreamVersion": "v22.0",
+  "version": "0.2.0",
```

Comment:
revert so we can do version bump properly and apply semver in another PR, the core team will handle it.